### PR TITLE
gr-path: add log for number of valid reads visited when making a silver path

### PR DESCRIPTION
**Pull request by jwcodee**
This pull request adds log information for how many valid reads were visited after generating x silver paths

### CODE REVIEW

**Comment by jwcodee** [comment]
https://github.com/bcgsc/goldrush/blob/d39cb4cf620d9042a28b672dc81d3a2c51a2c467/goldrush_path/goldrush_path.cpp#L793
@lcoombe  sure, let's enable verbose by default
While we are on that note, should I add this to verbose or debug. It's there to show that goldrush is still running, but I dont think it's that useful from a verbose perspective,

**Comment by lcoombe** [comment]
I think we keep that as default/verbose? I like seeing it to see that GoldRush is progressing through the reads?

**Comment by jwcodee** [comment]
Cool, Ill keep it as default then and ill push the verbose change soon too.

File: goldrush_path/goldrush_path.cpp [hunk 1]
```diff
@@ -59,9 +59,13 @@ silver_path_check(
   uint64_t& inserted_bases,
   uint64_t& curr_path,
   uint32_t& ids_inserted,
-  MIBFConstructSupport<uint32_t, multiLensfrHashIterator>& miBFCS)
+  MIBFConstructSupport<uint32_t, multiLensfrHashIterator>& miBFCS,
+  uint32_t& valid_reads)
 {
   if (target_bases < inserted_bases) {
+    if (opt::verbose) {
+      std::cerr << "Visited " << valid_reads << " reads" << "to generate " << curr_path << "silver paths" << std::endl;
```
Comment (lcoombe) [inline goldrush_path/goldrush_path.cpp:67]:
```suggestion
      std::cerr << "Visited " << valid_reads << " reads " << "to generate " << curr_path << " silver paths" << std::endl;

```

**Review by lcoombe — APPROVED** [review]
Thanks Johnathan!

**Pull request merged**